openapi/openapi.yaml

```diff
@@ -209,6 +209,39 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Scripts'
+
+  /backplane/script/{clusterId}:
+    get:
+      operationId: getScripts2
```

Comment:
The name here looks a bit weird... I am not quite familiar with the OpenAPI convention, but it should be better if we name this as something like `getScriptsByCluster`
Correct me if I am understanding wrong...

Reply:
I agree, the name is not ideal, I couldn't come up with a better one at the time.
I can change it to getScriptsByCluster, seems like a good fitting name.

Reply:
Other GET paths like `/backplane/cloud/credentials/{clusterId}` (or `/backplane/cloud/console/{clusterId}`) do not have the `operationId` suffixed with `ByCluster`.

If we don't like the `2` suffixing, what about:
- Renaming `getScripts` to `getScriptsOld`
- Creating the new path with the `operationId` set to `getScripts`?

Then once, the migration is finished, a PR would remove the old path with `operationId` set to `getScriptsOld`.

Remark that renaming an `operationId` is acceptable because:
- This id is not used in the server code.
- This id is used in the CLI code... but the use of the API is versioned:  
https://github.com/openshift/backplane-cli/blob/main/go.mod#L18. 
Also we have to modify CLI code anyway (to add the code for the new path); with that in mind we could sed the code to replace `GetScripts` with `GetScriptsOld` prior adding the code for the new path in the following locations:
https://github.com/search?q=repo%3Aopenshift%2Fbackplane-cli+getScripts&type=code